selenium/codenvy-selenium-test/src/test/java/com/codenvy/selenium/ssh/LoginBySshKeyTest.java

```diff
@@ -127,8 +125,6 @@ public void loginBySshKeyTest() throws Exception {
     String commandForConnection = getCommandFromSshConsole();
     ide.open(ws2);
     projectExplorer.waitProjectExplorer();
-    notifications.waitExpectedMessageOnProgressPanelAndClosed(
```

Comment:
could you, please, explain why this verification is removed? 
We need to ensure workspace 2 started correctly after all, don't it?

Reply:
Because this is extra check, we check that workspace is started by waiting project explorer, and if something went wrong next steps will not pass. Besides, waiting this pop-up message doing this test more unstable.

Reply:
Waiting of pop-up allows us to ensure that workspace COMPLETELY ready for testing.
Otherwise it doesn't make sense to continue testing, IMHO.

Reply:
But you are right - we can leave only one verification here.

Reply:
thanks for your comments

Reply:
you are welcome :-)